src/content/docs/en/guides/cms/strapi.mdx

```diff
@@ -350,23 +346,23 @@ try {
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>{article.attributes.title}</title>
+    <title>{article.title}</title>
   </head>
 
   <body>
     <main>
-      <img src={import.meta.env.STRAPI_URL + article.attributes.image.data.attributes.url} />
+      <img src={import.meta.env.STRAPI_URL + article.image.data.url} />
```

Comment:
Here's the other place with image data. I'll see if I can confirm in Strapi's docs.

Reply:
OK, found a [code sample example](https://docs.strapi.io/cms/api/rest/guides/understanding-populate#example-with-populate) here that seems to use `image.data.url` and no mention of an `attributes` level, so I'm going with this!